subsys/tracing/include/tracing_test.h

```diff
@@ -31,7 +31,12 @@ void sys_trace_isr_exit_to_scheduler(void);
 void sys_trace_idle(void);
 void sys_trace_void(unsigned int id);
 void sys_trace_end_call(unsigned int id);
-
+void sys_trace_semaphore_init(struct k_sem *sem);
```

Comment:
jw - I expected to see `sys_trace_start_call()` here (and ctf_top.c/h), too similar to the sys_trace_end_call, etc.. Looks like it hasn't been there yet but things work. Is it missing? Please correct me if I'm wrong.

Reply:
yeah, looks like sys_trace_start_call is missing in the test variant, this is just a test, no harm :)